--- openafs-1.5.66/aclocal.m4~	2009-10-23 21:57:33.000000000 +0000
+++ openafs-1.5.66/aclocal.m4	2009-10-27 12:44:38.000000000 +0000
@@ -11,11 +11,6 @@
 # even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.
 
-m4_if(m4_PACKAGE_VERSION, [2.61],,
-[m4_fatal([this file was generated for autoconf 2.61.
-You have another version of autoconf.  If you want to use that,
-you should regenerate the build system entirely.], [63])])
-
 # Copyright (C) 2002, 2003, 2005, 2006  Free Software Foundation, Inc.
 #
 # This file is free software; the Free Software Foundation
