--- openafs-1.2.10/src/venus/kdump.c.orig	Tue Jul  1 21:21:35 2003
+++ openafs-1.2.10/src/venus/kdump.c	Wed Oct  1 14:19:18 2003
@@ -213,7 +213,7 @@
 #include <ufs/ufs/inode.h>
 #include <ufs/ffs/fs.h> 
 #else
-#include "sys/vfs.h"
+//#include "sys/vfs.h"
 #ifdef AFS_LINUX20_ENV
 #define UIO_MAXIOV 1 /* don't care */
 #if __GLIBC_MINOR__ == 0
@@ -2566,8 +2566,8 @@
     printf("\ti_op=0x%x, i_dev=0x%x, i_rdev=0x%x, i_sb=0x%x\n",
 	   vep->i_op, vep->i_dev, vep->i_rdev, vep->i_sb);
 #ifdef AFS_LINUX24_ENV
-    printf("\ti_sem: count=%d, sleepers=%d, wait=0x%x\n",
-	   vep->i_sem.count, vep->i_sem.sleepers, vep->i_sem.wait);
+    printf("\ti_sem: count=%d, wait=0x%x\n",
+	   vep->i_sem.count, vep->i_sem.wait);
 #else
     printf("\ti_sem: count=%d, waking=%d, wait=0x%x\n",
 	   vep->i_sem.count, vep->i_sem.waking, vep->i_sem.wait);
--- openafs-1.2.10/src/venus/Makefile.in.orig	Tue Jul  1 21:04:15 2003
+++ openafs-1.2.10/src/venus/Makefile.in	Wed Oct  1 14:44:09 2003
@@ -117,7 +117,7 @@
 		${INSTALL} -s -f $? $@ ;; \
 	*linux* ) \
 		${INSTALLex} -f kdump.sh.linux ${DEST}/etc/kdump; \
-		${INSTALL} -s $? $@ ;; \
+		${INSTALL} -s -f $? $@;; \
 	hp_ux11* ) \
 		${INSTALLex} -f kdump.sh.hp_ux11 ${DEST}/etc/kdump; \
 		${INSTALL} -s -f $? $@;; \
